deepwell-core/Cargo.toml

```diff
@@ -17,11 +17,16 @@ workspace = ".."
 name = "deepwell_core"
 path = "src/lib.rs"
 
+[features]
+default = ["ftml_compat"]
```

Comment:
Feature with name '`ftml`' is more canonical. There might be an issue with name conflicts? If so ignore this.

Reply:
I'm worried about name conflicts with that.

Reply:
So the compilation fails if the names are the same? Okay